tools/system_libs.py

```diff
@@ -1687,6 +1695,9 @@ def add_library(lib):
     else:
       add_library(system_libs_map['libsockets'])
 
+  if shared.Settings.USE_WEBGPU:
+    add_library(system_libs_map['libwebgpu_cpp'])
```

Comment:
I think this should go on line 1681 (which is indented in)

Reply:
That's in the middle of sanitization stuff so I left it here but indented it properly.